### ISSUE TITLE
Elongated femoral neck

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7031]
**Preferred term label:**
Elongated femoral neck

**Synonyms**
Long femoral neck

**Definition (free text, please give PubMed ID)**
Increased length of the neck of the femur.

**Parent term (use hpo.jax.org/app)**
Abnormal femoral neck morphology

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
619345 
PMID: 33252156  1/3

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
